scripts/travis/build_test.sh

```diff
@@ -9,6 +9,8 @@
 # Examples: scripts/travis/build_test.sh
 set -e
 
+ALGORAND_DEADLOCK=enable
+export ALGORAND_DEADLOCK
```

Comment:
Nit: just `export ALGORAND_DEADLOCK=enable`

Reply:
I think that shellcheck doesn't like when you do that..

Reply:
Is that true for variable assignments or just when there's command output trapping error codes? Either way it's fine.